AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -309,6 +319,20 @@ export class SummaryComponent implements OnInit, OnDestroy {
         }
     }
 
+    async oberserableTimer(hearingId: string): Promise<UpdateBookingStatusResponse> {
+        let hearingStatusResponse;
+
+        const source = timer(0, 5000);
+        const schedule = source.subscribe(async counter => {
+            hearingStatusResponse = await this.hearingService.getStatus(hearingId);
+            if (hearingStatusResponse.success || counter === 6) {
```

Comment:
what is the 6 for here - is that the timeout?

if so we'll need to poll for longer than 30 seconds, as large bookings can take longer.

maybe go for 50 seconds, as per the ticket

Reply:
changed to 20 as any 5 seconds for 10 times is 50s